new-york-city.md

```diff
@@ -19,6 +19,8 @@ Lunch options near the office:
 
 * [All'onda](http://allondanyc.com) – Union Square, Mexican
 * Bryant Market - 38th and 6th Ave, Bagels
+* [Bryant Park Grill](http://www.bryantpark.org/things-to-do/bryant_park_grill.html) –
```

Comment:
The open table res link is good too. The other links seem to prefer the business's main website? Rather than 3rd party sites?